archethic-foundation/archethic-node
# Bootstrap: Replicate missed transaction of current summary

# Description

When a node starts and is authorized, it'll catch up the missed transaction of current summary (current day)
Fixes #1468

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Unit tested 
Manual test of this scenario: 

1. 4 nodes running
2. close one
3. run a faucet
4. restart the one
5. check that faucet transaction is in db

# Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules
